src/Features/Core/Portable/GenerateConstructorFromMembers/AbstractGenerateConstructorFromMembersCodeRefactoringProvider.cs

```diff
@@ -202,7 +202,11 @@ public async Task<ImmutableArray<IntentProcessorResult>> ComputeIntentAsync(Docu
             // Find all the possible writable instance fields/properties.  If there are any, then
             // show a dialog to the user to select the ones they want.  Otherwise, if there are none
             // don't offer to generate anything.
-            var viableMembers = containingType.GetMembers().WhereAsArray(IsWritableInstanceFieldOrProperty);
+            // First suggest `base` members, then `this` members.
+            var viableMembers = containingType.GetAccessibleMembersInBaseTypes<ISymbol>(containingType)
+                .Concat(containingType.GetMembers())
+                .Where(IsWritableInstanceFieldOrProperty)
+                .ToImmutableArray();
```

Comment:
i'm not sure how the logic is working here.  this is saying "find me writable fields/props in my bases", but:

1. nothing indicates that that means we can delegate all those to the base constructor (or is something different happening).
2. just because soemthing is writable *by the base* does not mean it's writable by a derived type.  For example, a `private set;` property isn't writable by a derived type.

Reply:
Since we are showing the user a list of fields, this probably cannot be done perfectly - different base constructors can cover different fields. What if we instead displayed all fields and only filtered the inaccessible ones when generating the code: 92ae5e33ffc4a27866a9f73599a5e3a2f3f0543e?